radio/src/gui/128x64/radio_diaganas.cpp

```diff
@@ -56,6 +56,24 @@ void menuRadioDiagAnalogs(event_t event)
 #endif
   }
 
+  // SWR
+#if defined(PCBX7)
+  if(IS_MODULE_XJT(EXTERNAL_MODULE) && !IS_INTERNAL_MODULE_ON()) {
+    coord_t y = MENU_HEADER_HEIGHT + 1 + (NUM_STICKS+NUM_POTS+NUM_SLIDERS+1)/2 * FH + 1 * FH + 2;
+      
+    lcdDrawText(1, y, "RAS:");
+    lcdDrawNumber(1 + 4*FW, y, telemetryData.swr.value, LEFT);
+  }
```

Comment:
But isn't RAS always invalid on X7 XJT as it does not have the measurement circuit?